README.md

```diff
@@ -36,40 +36,52 @@ Next, you must publish the config file:
 php artisan vendor:publish --provider="Spatie\FailedJobMonitor\FailedJobMonitorServiceProvider"
 ```
 
-This is the contents of the configuration file. Most options are self-explanatory.
+This is the contents of the default configuration file.  Most options are self-explanatory.
 
 ```php
 return [
```

Comment:
to make it clear to the users what this config is for, add some comments.

You can use [these lines](https://github.com/spatie/laravel-backup/blob/master/config/laravel-backup.php#L79-L82) for the `laravel-backup` package.